packages/tools/devtools/devtools-core/src/messaging/devtools-messages/CloseDevtools.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IDevtoolsMessage } from "../Messages";
+
+/**
+ * Encapsulates types and logic related to {@link CloseDevtools.Message}.
+ *
+ * @internal
+ */
+export namespace CloseDevtools {
+	/**
+	 * {@link DevtoolsFeatures.Message} {@link IDevtoolsMessage."type"}.
+	 *
+	 * @internal
+	 */
+	export const MessageType = "CLOSE_DEVTOOLS";
+
+	/**
+	 * Inbound message boradcasting the devtools is closed.
+	 * Will result in the {@link CloseDevtools.Message} message being posted.
```

Comment:
This is a message we post from Devtools "to the outside world" and results in no responses (in the form of messages) from anyone, so I think this is accurate.

```suggestion
	 * Outbound message broadcasting that the devtools instance in the webpage is terminating.
```